File: cmstestsuite/tasks/communication_stdio_stubbed/code/stub.py
import sys

import communication  # Submission format is ["communication.%l"]


while True:
    n = sys.stdin.readline().strip()
    if n == "":
        # EOF, let's quit.
        break

    n = int(n)
    if n == 0:
        break
    sys.stdout.write("correct %d\n" % communication.userfunc(n))
    sys.stdout.flush()

File: cmstestsuite/tasks/communication_stdio_stubbed/code/stub.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class stub {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        while (true) {
            int n = Integer.parseInt(br.readLine());
            if (n == 0)
                break;
            System.out.println("correct " + communication.userfunc(n));
            System.out.flush();
        }
    }
}
